### DISCUSSION TITLE
Third Party Calculated Shipping Rates - API support

### DISCUSSION THREAD

**Discussion by kiernangs** [etsy/open-api#1101] [Ideas]
Etsy currently only supports calculated rates in a few countries, and in the USA only with  USPS.

This is extremely limiting, especially as etsy expands its international footprint and international sales increase for us and canadian sellers.

Having etsy able to make calls to apps/APIs with live rate features would dramatically improve etsy competitively with shopify, etc that offer these functions. 

Some examples: 
Easypost - https://www.easypost.com/  and front end Shipblink -> https://shipblink.com/

or 

Shippo -  https://goshippo.com/

----
Both of these apps already offer api access for live rates to shopify and big commerce.

**Comment by emergingdzns** [comment]
@kiernangs Etsy uses Aftership for their shipping calculation and tracking. If the carriers you want are integrated (via api) with Aftership you can ask Etsy to enable it. They may be able to do it.

**Reply by kiernangs** [reply]
I have tried to make these requests repeatedly. They mostly don't.  Unclear why, as they don't really send anything other than form letter replies.

But that is also not what this suggestion is.  Aftership provides tracking.

I am talking about Live Shipping Rates.  As in calculated rates from carriers other than USPS. Specifically calculated rates that arent just the retail rates, but are negotiated rates for the specific seller or platform.
